libs/elasticsearch/README.md

```diff
@@ -192,4 +192,58 @@ set_llm_cache(
 ```
 
 When overriding the mapping and the document building, 
-please only make additive modifications, keeping the base mapping intact.
\ No newline at end of file
+please only make additive modifications, keeping the base mapping intact.
+
+###  ElasticsearchEmbeddingsCache
+
+Store and temporarily cache embeddings.
+
+Caching embeddings is obtained by using the [CacheBackedEmbeddings](https://python.langchain.com/docs/modules/data_connection/text_embedding/caching_embeddings),
+it can be instantiated  directly as an argument of the `CacheBackedEmbeddings` class or using `from_bytes_store` method.
```

Comment:
The example below still shows the old way of initializing. Did you want to adapt this now that we have a ByteStore?

Reply:
ok